### ISSUE TITLE
Loading an sf2 file causes audible click

### ISSUE THREAD

**Issue by MjnMixael** [melihhakanpektas/flutter_midi_pro#24]
As it says, when loading an sf2 file the packages creates an audible click sound. It's not major, but it's unideal for the user experience.

This is the code that reproduces the problem.

```
void initSoundFont() async {
  soundFontId = await MidiPro().loadSoundfont(path: 'assets/SquareWave.sf2', bank:0, program: 0);
}
```

**Comment by cddqssc** [comment]
Encountered the same problem

**Comment by melihhakanpektas** [comment]
Hey flutter developers, this bug was fixed on version of 3.1.0. Thank you for your comments :)
